app/views/partials/ModulesPartial.scala.html

```diff
@@ -21,7 +21,9 @@
 )
 
 @(
-  repositoryModules: RepositoryModules
+  repositoryModules : RepositoryModules,
+  isLatest          : Boolean         = false,
```

Comment:
wonder if naming `isLatest` as `displayBuildInfo` will decouple it from domain knowledge, and make it easier to understand why it's being asked for?